Assets/Scripts/BowlingBall.cs

```diff
@@ -1,151 +0,0 @@
-﻿using System;
-using System.Collections;
-using System.Collections.Generic;
-using System.Security.AccessControl;
-using UnityEngine;
-using UnityEngine.EventSystems;
-using UnityEngine.Experimental.UIElements;
-using UnityEngine.XR.WSA;
-
-public class BowlingBall : MonoBehaviour
-{
-
-	// Use this for initialization
-	public Vector3 InitialVelocity = Vector3.forward;
-	public Vector3 InitialPosition;
-	public float TranslationCoefficient;
-
-	private bool isMoving;
-	private Rigidbody ballRigidbody;
-	private AudioSource audioSource;
-	private string translationDirection;
-	private const float LANE_BOUNDS = 0.45f;
-	private Vector3 GUTTERBALL_BOUNDS = new Vector3(1, -1, 27);
-	private bool readyToLaunch;
-	private bool hasLaunched;
-	private bool ballEnteredBox;
-
-
-
-
-	void Start()
-	{
-		ballRigidbody = GetComponent<Rigidbody>();
```

Comment:
Good caching components in start, to avoid the cost of doing it every update.

If you're MonoBehaviour absolutely has to have a RigidBody to function, it's best practice to specify that on your Behaviour so unity will warn users and enforce this for you.

```
[RequireComponent(typeof(RigidBody))]
public class BowlingBall : MonoBehaviour
{
   ...
}
```